curvefs/proto/common.proto

```diff
@@ -23,12 +23,21 @@ enum FSType {
     TYPE_S3 = 2;
 }
 
+enum BitmapLocation {
+    AtStart = 1;
+    AtEnd = 2;
+}
+
 message Volume {
     required uint64 volumeSize = 1;
     required uint64 blockSize = 2;
     required string volumeName = 3;
     required string user = 4;
     optional string password = 5;
+    required uint64 blockGroupSize = 6;
+    required BitmapLocation bitmapLocation = 7;
+
+    // TODO(all): maybe we need curvebs cluster's mds ip and port in here
```

Comment:
> Didn't you say that you have used other learning interfaces before?

it's because, for unaligned write operation, RMW is performed, so there might be a data consistent problem, like https://github.com/opencurve/curve/issues/985. and if this really happens, we should redesign the implementation here. so I just add a dedicated thread pool now.

> Can we change from AtStart to AtEnd?

yep, both are supported, but they have no much difference.

> Maybe we can use the id to mark the curvebs cluster. The correspondence between the id and the curvebs cluster is recorded in the curvefs-mds.

currently, it's only affects how we extend the volume, now we can only add an extra client.conf in mds.

and if we are going to support volume in different curvebs clusters, we may need ip/port or id.